fury/shaders/billboard_dec.vert

```diff
@@ -1,4 +1,5 @@
 /* Billboard  vertex shader declaration */
 in vec3 center;
+
 out vec3 centeredVertexMC;
 out float scalingFactor;
```

Comment:
and this would be `scalingFactorVSOutput`

Reply:
I have a question regarding this variable. We are passing this variable to the fragment shader however it is not used in any of the current examples we have. Should we restrict its usage only to the vertex shader code?

Reply:
If it isn't used anywhere, then it is probably safe to stop passing it through. Perhaps it is just left over from some experiment and isn't really needed.